### DISCUSSION TITLE
Is it possible to move the error block (vf-errors) to the bottom of the form near the submit button?

### DISCUSSION THREAD

**Discussion by djkunkel** [vueform/vueform#125] [Questions] [answered]
We have some longer forms created with the builder, and the error messaging being at the top of the form is off the screen for users. Is there a way I'm not seeing to get it to render this list at the bottom of the form? I was able to set display-errors to false to hide it, but I'd still like users to get submission errors if there are any.

**Comment by djkunkel** [comment]
I'm currently solving this by creating my own theme and creating a new template for Vueform (copied from blank template) that moves the FormMessages and FormErrors components to the bottom of the template.

**Reply by adamberecz** [reply, marked as answer]
This is the right approach, except you don't necessarily need a new theme, you can register it as an [alternative view](https://vueform.com/docs/styles-and-layout#alternative-views) for Vueform.

Eg. `Vueform_endErrors` and use it like:
```vue
<Vueform view="endErrors">
```

**Reply by djkunkel** [reply]
Thank you! I missed this section in the docs.

**Reply by adamberecz** [reply]
It was recently added only ;)
